rambo/vagrant_resources/vagrant/core

```diff
@@ -52,3 +52,51 @@ Vagrant.configure("2") do |config|
     puts "Vagrant done with destroy."
   end
 end
+
+#redundant code from LXC and VirtualBox vagrant files
+Vagrant.configure("2") do |config|
+  if COPY_DIR_WITH_RSYNC == true
+    config.vm.synced_folder ".",
+      "/vagrant",
+      type: "rsync"
+  else
+    config.vm.synced_folder ".",
+      "/vagrant",
+      type: "rsync",
+      disabled: true
+  end
+  config.vm.provision "shell",
+    inline: "wget -O /etc/apt/sources.list https://raw.githubusercontent.com/terminal-labs/package-sources/master/" + VM_HOST + "/official/sources.list",
+    keep_color: true
+  if PROVISION_WITH_SALT
+    config.vm.provision :salt do |salt|
+      salt.bootstrap_options = "-P"
+     salt.verbose = true
+    end
+    $pre_salt = <<-EOS
+      mv /etc/salt/minion{,-dist}
+      if [ target == "virtualbox" ]; then cp /vagrant/salt_resources/minions/minion.virtualbox /etc/salt/minion; fi
```

Comment:
This if chunk (through line 85) should be replaced with a single line that just uses the value of `target` directly. There's no need for the string check.